.devcontainer/Dockerfile

```
@@ -1,5 +1,9 @@
 FROM mcr.microsoft.com/devcontainers/typescript-node:1-22-bookworm
 
+# Setup support for Git LFS
+RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash &&\
```

Hadolint

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)